libtfhe/core/arithmetic/torus_utils.h

```diff
@@ -38,18 +43,18 @@ class TorusUtils {
      * @param d real to convert
      * @return torus value
      */
-    static TORUS from_double(double d) {
+    static void from_double(TORUS &reps, const double d, const ZModuleParams<TORUS> *params) {
```

Comment:
keep the two variants of the function: 
return TORUS and take TORUS as argument

Reply:
For BigTorus instantiation we'll need the allocator if we want to return a TORUS. If both functions are desirable I'll add the allocator as function parameter.

Reply:
Makes sense, but will loose "genericity" of the code using the return by value variant.